src/pages/index.js

```diff
@@ -154,15 +256,19 @@ profileEditButton.addEventListener("click", () => {
 });
 
 cardModalBtn.addEventListener("click", () => {
+  resetValidation(cardForm, [cardNameInput, cardLinkInput], settings);
```

Comment:
`  resetValidation(cardForm, [cardNameInput, cardLinkInput], settings);
`

Please, delete it 


`resetValidation` is for clearing possible validation errors when you open the **profile** form, because you insert **valid** data into its inputs. But you don't need to clear errors for the card and avatar forms, according to the task


 It’s a **waste of resources**.